369905 first pass create python script to populate sequence panels from flix 5 (#4)
TheFoundryVisionmongers/flix-api-sdk: README.md

After being logged in, you will have a token.
This token will be use to sign your requests.

This is the signature of the function from `flix6/signature.py`

```fnSign(access_key_id, secret_access_key, url, content, http_method, date, content_type):```

    `access_key_id`: Access key ID from your token

    `secret_access_key`: Secret access key from your token

    `url`: Url of the request

    `content`: Content of your request

    `http_method`: Http Method of your request

    `content_type`: Content Type of your request

## Migration tool

> The `migrate5to6.py` is a script that allow you to list or migrate a show / episode / sequences / sequences revisions in Flix 6.

Reviewer: ```suggestion
The `migrate5to6.py` is a script that allows you to list or migrate shows, episodes, sequences and/or sequences revisions from Flix 5 into Flix 6.
```

Required: python3

```shell
usage: migrate5to6.py [--help] [--server SERVER] [--user USER]
                      [--password PASSWORD] [--flix-projects FLIX_PROJECTS]
                      (--list | --migrate) [--shows [SHOWS [SHOWS ...]]]
                      [--episodes [EPISODES [EPISODES ...]]]
                      [--sequences [SEQUENCES [SEQUENCES ...]]]
                      [--all-revisions] [--port PORT]

optional arguments:
  --help
  --shows [SHOWS [SHOWS ...]]
                        Shows to use
  --episodes [EPISODES [EPISODES ...]]
                        Episodes to use
  --sequences [SEQUENCES [SEQUENCES ...]]
                        Sequences to use
  --all-revisions       All sequence revisions
  --port PORT           Download helper port

required arguments:
  --server SERVER       Flix 6 server url
  --user USER           Flix 6 client username
  --password PASSWORD   Flix 6 client password
  --flix-projects FLIX_PROJECTS
                        Flix 5 projects path
  --list                Print the list
  --migrate             Migrate / Import into Flix 6
```

## Example

> This command will list all the informations from the show `my_show`

Reviewer: ```suggestion
This command will list all the information from the show `my_show`.
```

```shell

> python3 migrate5to6.py --server http://localhost:1234 --user admin --password admin --flix-projects ~/Desktop/flix5Projects --shows my_show --list

Reviewer: ```suggestion
python3 migrate5to6.py --server http://my-flix-server:8080 --user admin --password admin --flix-projects ~/Desktop/flix5Projects --shows my_show --list
```
